src/py/flwr/cli/new/templates/app/pyproject.flowertune.toml.tpl

```diff
@@ -30,10 +30,37 @@ serverapp = "$import_name.app:server"
 clientapp = "$import_name.app:client"
 
 [tool.flwr.app.config]
-num-server-rounds = 3
+model.name = "mistralai/Mistral-7B-v0.3"
+model.quantization = 4
+model.gradient_checkpointing = true
+model.lora.peft_lora_r = 32
+model.lora.peft_lora_alpha = 64
+train.save_every_round = 5
+train.learning_rate_max = 5e-5
+train.learning_rate_min = 1e-6
+train.seq_length = 512
+train.training_arguments.output_dir = ""
+train.training_arguments.learning_rate = ""
+train.training_arguments.per_device_train_batch_size = 16
+train.training_arguments.gradient_accumulation_steps = 1
+train.training_arguments.logging_steps = 10
+train.training_arguments.num_train_epochs = 3
+train.training_arguments.max_steps = 10
+train.training_arguments.save_steps = 1000
+train.training_arguments.save_total_limit = 10
+train.training_arguments.gradient_checkpointing = true
+train.training_arguments.lr_scheduler_type = "constant"
+strategy.fraction_fit = $fraction_fit
+strategy.fraction_evaluate = 0.0
+num-server-rounds = 200
```

Comment:
previously the number of rounds was in the `static_config.yaml`. So people can adjust it now? it's fine with me to let them do so~

Reply:
Yeah, as we discussed previously, we don't fix the number of rounds.